warn/scrapers/mi.py

```diff
@@ -1,6 +1,7 @@
 import re
```

Comment:
If this is a Michigan scraper, it should be in a separate pull request from the WV scraper. Let's do these one at a time.

Reply:
Michigan is being removed again. Can we take out any edits to it entirely?

Reply:
*reply to picture down below.
My bad, I think I forgot to add mi.py to this one. Sorry for the late fix, I got the flu again😐

Reply:
Great. Thanks.